### PR TITLE
Swallow some errors

### PR DESCRIPTION
**Pull request by fregante**
- Closes https://github.com/pixiebrix/pixiebrix-extension/issues/2622

This completely swallows the errors: They're not logged and they're not sent to rollbar. Is this what you're looking for? I haven't been able to reproduce that Cancel error though.

### CODE REVIEW

File: src/devTools/Panel.tsx [hunk 1]
```diff
@@ -72,17 +71,6 @@ const Panel: React.FunctionComponent = () => {
         <div>{error}</div>
         <div className="mt-2">
           <Button
-            onClick={() => {
```
Comment (twschiller) [inline src/devTools/Panel.tsx:75]: @fregante Should this change be in here? Seems related to https://github.com/pixiebrix/pixiebrix-extension/pull/2587

Comment (fregante) [reply src/devTools/Panel.tsx:75]: heh too many PRs open 😅

**Review by twschiller — APPROVED** [review]
Conditional approval: See comment about unrelated change

The "Promise Cancelled" error comes up with a menu extension point is waiting for the menu to appear on the page (using awaitElementOnce) but you navigate to a different page in the SPA

**Comment by fregante** [comment]
Wrong commit moved to https://github.com/pixiebrix/pixiebrix-extension/pull/2587

**Pull request merged**